docs/learn/calls.md

```diff
@@ -174,6 +184,8 @@ response = recommend_book("fantasy")
 print(response.content)
 ```
 
+In this example, we're creating a custom OpenAI client with a specific API key, organization ID, and base URL. This custom client is then passed to the call decorator, allowing us to use a custom endpoint or configuration for our LLM calls.
+
 Any custom client is supported so long as it has the same API as the original base client.
 
 ## Handling Responses
```

Comment:
nit: as mentioned in the review on the "Prompts" docs, things like "metadata" and the link will need to be updated to point to the new section since it'll no longer live in the prompts docs.